### DISCUSSION TITLE
Status for paused/postponed/on hold

### DISCUSSION THREAD

**Discussion by ChemicalXandco** [jotaen/xit#42] [File Format]
For some items it makes sense to mark them as paused because they have been started but are not being worked on in the short term. It does not make sense to keep them as ongoing because this does not differentiate from tasks being actively worked on. It does not make sense to change to obsolete because that means it is not going any further. I believe this requirement justifies another status.

I suggest the syntax for such a status should be `[.]` or also consider `[p]`

**Comment by jotaen** [comment]
Cool, thanks for your idea. I’ve [opened an issue](https://github.com/jotaen/xit/issues/44) to collect ideas for adding more statuses, and added yours there.
